.zuul.yaml

```diff
@@ -88,7 +88,6 @@
       - name: openstack-k8s-operators/ci-framework
         override-checkout: main
       - name: github.com/infrawatch/service-telemetry-operator
-        override-checkout: master
       - name: github.com/infrawatch/smart-gateway-operator
         override-checkout: master
```

Comment:
For testing stable-1.5 when it is merged, we can have `stable-1.5` in these override-checkout.

For testing before the changes are all backported to stable-1.5, we'll need to reference the proposed branches.

I recommend the following approach:
* someone created a PR to merge master into stable-1.5 (starting the release process)
* You propose the updates you identified here, targetting the PR branch, using `override-checkout: <PR_branch_name>`
* Once zuul tests are passing on the zuul-stable15-pr branch, release-branch gets merged
* Once release-branch is merged, zuul-stable-15-pr gets updated to use `override-checkout: stable-1.5`